frontend/packages/gitops-plugin/src/components/list/GitOpsTableRow.tsx

```diff
@@ -41,12 +119,65 @@ const GitOpsTableRow: RowFunction<GitOpsAppGroupData> = (props) => {
           <span style={{ marginRight: 'var(--pf-global--spacer--xs)' }}>{appGroup.repo_url}</span>
         </ExternalLink>
       </TableData>
-      <TableData className={tableColumnClasses[2]}>{appGroup.environments.join(', ')}</TableData>
+      <TableData className={classNames(tableColumnClasses[2], 'pf-u-text-nowrap')}>
+        {appGroup && appGroup.sync_status ? (
+          <Flex className="syncStatus">
+            <GetSyncFragment tooltip={ttSynced} count={numOfSynced} icon="check" />
+            <GetSyncFragment tooltip={ttOutOfSync} count={numOfOutOfSync} icon="exclamation" />
+            <GetSyncFragment tooltip={ttUnknown} count={numOfUnknown} icon="unknown" />
+          </Flex>
+        ) : (
+          <span>{appGroup.environments.join(', ')}</span>
+        )}
+      </TableData>
       <TableData className={tableColumnClasses[3]}>
-        {/* this is just a placeholder until backend changes can go in to get this data */}-
+        {latestDeployedTime ? (
+          <Flex>
+            <FlexItem className="lastDeploymentTime" spacer={{ default: 'spacerXs' }}>
+              <span>
+                <Timestamp timestamp={latestDeployedTime} />
+              </span>
+            </FlexItem>
+            <FlexItem>{latestDeployedEnv}</FlexItem>
+          </Flex>
+        ) : (
+          <span>-</span>
+        )}
       </TableData>
     </TableRow>
   );
 };
 
+const GetSyncFragment: React.FC<SyncProps> = ({ tooltip, count, icon }) => {
+  let targetIcon: any;
+  if (icon === 'check') {
+    targetIcon = <GreenCheckCircleIcon />;
+  } else if (icon === 'exclamation') {
+    targetIcon = <YellowExclamationTriangleIcon />;
+  } else {
+    targetIcon = <GrayUnknownIcon />;
+  }
+  return (
+    <Flex flex={{ default: 'flex_1' }}>
+      <FlexItem>
+        {count > 0 ? (
+          <Tooltip isContentLeftAligned content={<div>{tooltip}</div>}>
+            <Flex>
+              <FlexItem>
+                {targetIcon} {count}
+              </FlexItem>
+            </Flex>
+          </Tooltip>
+        ) : (
+          <Flex>
+            <FlexItem>
+              {targetIcon} {count}
+            </FlexItem>
+          </Flex>
+        )}
+      </FlexItem>
+    </Flex>
+  );
+};
```

Comment:
I just noticed this... this is a lot of Flex for zero(?) Flex usage. I think you can achieve this with just divs. 

```suggestion
    <div> // may need to do a display: inline-block class here
      {count > 0 ? (
        <Tooltip isContentLeftAligned content={<div>{tooltip}</div>}>
          {targetIcon} {count}
        </Tooltip>
      ) : (
        {targetIcon} {count}
      )}
    </div>
  );
};
```

Reply:
I was able to make all 'three columns' (consisting of the icon and number) to align vertically across rows.  I had tried removing the outer Flex, but that didn't work.  I can try <div>

Reply:
You are likely suffering from the `display: block` problem. Flex likely go around that with the use of the flex styles. `display: inline-block;` should work... but if it does not, we can probably keep the outer Flex. The inner ones were unneeded.

Reply:
Hey Andrew, it didn't work.   I need the flex.  Also, I need an inner div or else there is an error.  eg. replace the flex/flexitem with a div

```
    <Flex flex={{ default: 'flex_1' }}>
      <FlexItem>
        {count > 0 ? (
          <Tooltip isContentLeftAligned content={<div>{tooltip}</div>}>
            <div>
              {targetIcon} {count}
            </div>
          </Tooltip>
        ) : (
          <div>
            {targetIcon} {count}
          </div>
        )}
      </FlexItem>
    </Flex>
```

Reply:
Ah fair, that works then.